### PR TITLE
Docker test

### PR DESCRIPTION
**Pull request by nonnib**
Switched to pypi packages for drift and drift-config instead of using git develop.
Added python-decouple to allow for easy env based configuration of the app to facilitate docker development and deployment.
Added Makefile for easy local development.
Remove celery since it has not been working for a long time and interferes with docker migration.
Temporarily removed marshmallow schema from an endpoint that expected data to be in another format than the schema.
This is the point of no return. This version will not be deployable using the old AMI build.

### CODE REVIEW

**Comment by judgeaxl** [comment]
So given this is point of no return, how do we run/upgrade the tier now?

File: .travis.yml [hunk 1]
```diff
@@ -2,37 +2,57 @@ sudo: true
...
     - AWS_DEFAULT_REGION=eu-west-1
     - AWS_DEFAULT_OUTPUT=json
     - DRIFT_TIER=DEVNORTH
-  matrix:
+    - PATH=$HOME/.local/bin:$PATH
+    - BOTO_CONFIG=/dev/null
     - DRIFT_CONFIG_URL=s3://relib-test/directive-games
+    - DOCKER_USERNAME=nonnib
+    - IMAGE_NAME=directivegames/drift-base
+    - secure: VW2DI3gVIPeZRwsyVHPOIEkRHWlCHr00TAhx0c/Y0OVnyFAQUA/XymNGbDf92/r8T/fQoKjdm9j9zp+XupNiDkMOH0y+uA1poolF/npnqvUIDGIkj67eF5HcyM1QCsie1iCm/JuyMCVFG+XQy6AfhyHyjvvGFMLxInADvgmsQMLYuLU/5ZdsLcv7fJcPGMTjxqtcEf017ZbSAShDYmag91jNdKOTU50C16c+m2dEskDM5JDhWvxeJ6mkdhmdtHRAIGkG4Ix+nd0dzBFhCd8V8TjUW/kp9XmAbjPkHbu6h3UKAph8bkW6O+wmHkEnmaDRsSuKTv6QFwUEZgzodacJ1azYjSXoigpKlTTGp4AxDxMD23qix8Wr1KxGIGKXRQSkLWaWym2G6Gri7NEHuJidgZQ/RQJA6k6aur5xB/bIdnGxtqmiv8sJlWuvw8HnyEMv1i8BViFgGh7CO1fpfrPokbi6wMjENc6ObmzoH6u+MeFyZyolco6eJ2JeivgvWUGnfitZPPuTsxg+gaUj4ei8D9ilUQZSKLf4dmrXkqrii5eRg+hdS68lno2kcZIPnMbh23m6KjrHofFanAfm/DOM0bcN0FtqLlrw4B2JOUYuB5lHnTCqJtAxEv3Rpx8DHG0VNzZEdW2Rjp5VdRa0Ea5hy63L7mPxHlIRKzRsMLPz+N4=
```
Comment (judgeaxl) [inline .travis.yml:16]: What's this? Secure what? This is a public repo. Please remove, *and* change it.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+build:
+	docker build -t ${IMAGE_NAME} .
+	docker tag ${IMAGE_NAME} ${IMAGE_NAME}:${BRANCH}
+
+git-tag:
+	git tag ${VERSION}		
+	git push origin --tags -o ci.skip
+
+run:
+	docker run -e DRIFT_TIER=DEVNORTH \
```
Comment (judgeaxl) [inline Makefile]: Why do we push deployment configuration data to a public repo?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+			   -e DRIFT_CONFIG_URL='redis://redis.devnorth.dg-api.com:6379/0?prefix=dgnorth' \
+			   -e DRIFT_DEFAULT_TENANT=dg-daedalus-devnorth \
```
Comment (judgeaxl) [inline Makefile]: Why would this be the default tenant? And why is it even here?

**Review by judgeaxl — CHANGES_REQUESTED** [review]
When this is merged, how is the tier updated? Also see security questions. I can't really comment on the individual changes of libraries and versions.

File: .travis.yml (hunk 1 above)
Comment (nonnib) [reply .travis.yml:16]: 'secure' is a travis paradigm for encrypting information and should be safe in a public repo. Some information here. https://docs.travis-ci.com/user/encryption-keys/

File: Makefile (hunk 3 above)
Comment (nonnib) [reply Makefile]: The makefile is a bit of a work in progress for easy development

**Comment by nonnib** [comment]
> So given this is point of no return, how do we run/upgrade the tier now?

We will move to a docker-only process this week was my thinking.

**Pull request merged**